### ISSUE TITLE
NTR: MRI: cranial nerve VI hypoplastic or absent (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8934]
New term request MRI: cranial nerve VI hypoplastic or absent (elthat_101022011204): Sixth nerve; 6th; abducens small  UMLS: 5393521

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000596
- label: Sixth cranial nerve hypoplasia
- parents: [Abnormal sixth cranial nerve morphology (HP:0011348)](https://hpo.jax.org/app/browse/term/HP:0011348)
- pmid: PMID:28613463
- def: Underdevelopment of the six cranial nerve (abducens nerve).
- comment: Cranial nerve six (CN VI) is one of the nerves responsible for the extraocular motor functions of the eye, along with the oculomotor nerve (CN III) and the trochlear nerve (CN IV).
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8934
- synonyms:
